packages/lit-html/src/lit-html.ts

```diff
@@ -323,8 +323,26 @@ const tag =
 export const html = tag(HTML_RESULT);
 
 /**
- * Interprets a template literal as an SVG template that can efficiently
+ * Interprets a template literal as an SVG fragment that can efficiently
  * render to and update a container.
+ *
+ * ```ts
+ * const rect = svg`<rect width="10" height="10"></rect>`;
+ *
+ * const myImage = html`
+ *   <svg viewBox="0 0 10 10" xmlns="http://www.w3.org/2000/svg">
+ *     ${rect}
+ *   </svg>`;
+ * ```
+ *
+ * The `svg` tagged template literal (TTL) should only be used for SVG
+ * fragments, or elements that would be contained within an `<svg>` html tag. A
+ * common error is wrapping `<svg>` with the svg TTL. The `<svg>` element is an
```

Comment:
When talking about `html`, we have generally called it a tag or tag function, and we haven't used TTL elsewhere in the docs, so I'd be inclined to stay with that. We could also say "a template literal tagged..." here but it seemed a little ungainly and this phrasing doesn't seem to ambiguous to me, but I could go either way.

```suggestion
 * common error is placing an `<svg>` *element* in a template tagged with the `svg` *tag function*. The `<svg>` element is an
```